### ISSUE TITLE
How to perform fine tuning for classification. 

### ISSUE THREAD

**Issue by MuhammadIrzam447** [NX-AI/vision-lstm#4]
I have been able to use the .yaml files in the pretrain directory to start the training on custom datasets. However, when I try to perform the fine tuning by using the example.yaml file at "/vision-lstm/src/vislstm/yamls/vil/pretrain/pooling/finetune" and it does not work. 

I am running the whole setup on a single GPU using the following command for pretraining
```
CUDA_VISIBLE_DEVICES=3 python main_train.py --devices 3 --hp vislstm/yamls/vil/pretrain/lstm_6M16_e400_bialter_bilatavg_lr1e3_res224.yaml
```

Meanwhile, the issue arises for the fine tuning. I am downloading the models and putting them in the "/vision-lstm/src/models/.." but for some reason it does not seem to work. 
The error originates from the "kwargs in model attributes in example.yaml" file. 

The values are as follows:
```
model:
  kind: models.single.vislstm
  patch_size: 16
  kwargs: ${select:tiny:${yaml:vision-lstm/src/models/vil_tiny16_e400_in1k}}
  mode: classifier
  pooling:
    kind: class_token
```

**Comment by BenediktAlkin** [comment]
The [provided fine-tuning yaml](https://github.com/NX-AI/vision-lstm/blob/main/src/vislstm/yamls/vil/pretrain/pooling/finetune/example.yaml) fine-tunes a model that was pre-trained on lower resolution on ImageNet-1K on 224x224 resolution by loading it from the save directory of the pre-training run. So the pre-training run will create a directory "OUTPUT_PATH/STAGE_NAME/STAGE_ID/checkpoints/CKPT_FILE.th" and in the fine-tuning yaml, you need to specify the STAGE_NAME, STAGE_ID and parts of the CKPT_FILE (via `model_name` and `ckpt`).

As far as i understand, you want to fine-tune an ImageNet-1K pre-trained ViL model on a custom dataset. For this, I added an [example](https://github.com/NX-AI/vision-lstm/blob/main/src/vislstm/yamls/vil/transfer/finetune_from_torchhub.yaml) how to fine-tune it on CIFAR10. As meantioned in the yaml, you probably need to play around with hyperparameters to get good results, i simply copied the hyperparameters from the "fine-tune to higher-resolution" setting.

The example yaml achieves an accuracy of 93.3% on CIFAR-10 after 20 epochs, but as I said, the hyperparameters are most likely suboptimal.

**Comment by MuhammadIrzam447** [comment]
Dear @BenediktAlkin. 

Thankyou for the quick response. You understand it right, for the time being, i wanted to investigate the fine-tuning of torch hub models of VIL on different custom datasets to compute their overall performance. 

However, as indicated in the examples datasets file e.g cifar10, cifar100 etc, my question is do we need to convert the custom dataset in the dataset format like cifar10 or it will also work with ImageNet dataset format. 

My question comes after a bit exploration into the model where i have managed to run the fine tuning for the cifar10 with the yaml file that you kindly shared in the last commit. however, problem arises when i try to use a custom data stored locally on the system.
